images/expansionhunter/Dockerfile

```diff
@@ -0,0 +1,50 @@
+
+FROM ubuntu:20.04
+ARG DEBIAN_FRONTEND=noninteractive
+ENV PATH="/root/miniconda3/bin:${PATH}"
+ARG CONDA_RELEASE="4.6.14"
+ARG CONDA_INSTALL_DIR="/opt/conda"
+ARG CONDA_BIN=${CONDA_INSTALL_DIR}/bin
+ARG CONDA_CMD=${CONDA_BIN}/conda
+ENV PATH=${CONDA_BIN}:$PATH
+ENV EH_VERSION=v5.0.0
+
+RUN apt-get update && apt-get install --no-install-recommends -qqy \
+    python3-dev \
+    python3-pip \
+    python \
+    python-dev \
+    awscli \
```

Comment:
Is the `awscli` needed for running code through expansion hunter, or would it be safe to remove?

Reply:
I have removed.